exercises/concept-wip/crystal-hunter/.meta/src/exemplar.cr

```diff
@@ -0,0 +1,17 @@
+class Rules
+  def bonus_points?(power_pellet_active, touching_bandit)
```

Comment:
Is there a reason that these are instance methods instead of class methods?

Reply:
It makes so we don't have to explain what self does. As fast as we reach the class concept, I think that should be explained, and that concept appears quite early.